File: class05/cpp/AP0502-CalculadoraSimples.cpp
#include <iostream>
#include <iomanip>
int main() {
    double a, b;
    std::cout << "N1: "; std::cin >> a;
    std::cout << "N2: "; std::cin >> b;
    std::cout << std::fixed << std::setprecision(2);
    std::cout << "Soma: " << a+b << " Sub: " << a-b << " Mult: " << a*b << " Div: " << a/b << std::endl;
    return 0;
}

File: class05/java/AP0502-CalculadoraSimples.java
import java.util.Scanner;
public class Calc {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("N1: "); double a = sc.nextDouble();
        System.out.print("N2: "); double b = sc.nextDouble();
        System.out.printf("Soma: %.1f Sub: %.1f Mult: %.1f Div: %.2f%n", a+b, a-b, a*b, a/b);
    }
}
